flower_classifier/callbacks/gradual_unfreezing.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from pytorch_lightning import LightningModule, Trainer
+from pytorch_lightning.callbacks import Callback
+
+logger = logging.getLogger(__name__)
```

Comment:
I'm not sure I understand the difference between this logger and the logger from the WandbLogger class.